docs-devsite/vertexai-preview.functiondeclaration.md

```diff
@@ -0,0 +1,57 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# FunctionDeclaration interface
+Structured representation of a function declaration as defined by the \[OpenAPI 3.0 specification\](https://spec.openapis.org/oas/v3.0.3). Included in this declaration are the function name and parameters. This FunctionDeclaration is a representation of a block of code that can be used as a Tool by the model and executed by the client.
```

Comment:
NOT BLOCKING I think that FunctionDeclaration in the second sentence should be tagged as code.

Note that I've only marked this instance, but it seems there are lots of places throughout the source code that should be tagged as code, like FunctionDeclarationsTool, FunctionCall, ChatSession, generateContentStream(), generateContent(), etc.